app/models/manageiq/providers/ibm_cloud/inventory/collector/vpc/target_collection.rb

```diff
@@ -0,0 +1,90 @@
+class ManageIQ::Providers::IbmCloud::Inventory::Collector::VPC::TargetCollection < ManageIQ::Providers::IbmCloud::Inventory::Collector::VPC
+  def initialize(_manager, _target)
+    super
+
+    parse_targets!
+  end
+
+  def images
+    []
+  end
+
+  def instances
+    @instances ||=
+      references(:vms).map do |ems_ref|
+        compute_client.get_instance(ems_ref)
+      end
+  end
+
+  def instance_types
+    []
+  end
+
+  def flavors
+    []
+  end
+
+  def auth_key_pairs
```

Comment:
This should be `keys`: https://github.com/ManageIQ/manageiq-providers-ibm_cloud/blob/master/app/models/manageiq/providers/ibm_cloud/inventory/collector/vpc.rb#L34